src/client.js

```diff
@@ -69,6 +68,15 @@ async function getOneDriveClient() {
   } catch (e) {
     // ignore
   }
+
+  let {
+    AZURE_APP_REFRESH_TOKEN: refreshToken = '',
+  } = process.env;
+
+  if (!refreshToken && tokens.length) {
+    refreshToken = tokens[0].refreshToken;
+  }
```

Comment:
If no `refreshToken` was specified, use a cached one if available. This changes the `AuthenticationContext` API being called from `acquireTokenWithClientCredentials` to `acquireTokenWithRefreshToken`